### ISSUE TITLE
How to make invisible recaptcha, Formik and yup work together?

### ISSUE THREAD

**Issue by MuriloScarpaSitonio** [dozoisch/react-google-recaptcha#199]
react-google-recaptcha version: 2.1.0
react-async-script version: 1.2.0

Hello!

I'm trying to make invisible recaptcha, [Formik](https://formik.org/docs/overview) and [yup](https://github.com/jquense/yup) to work together. The documentation says we should call `recaptchaRef.current.execute()` on form submission, but if we use Formik and yup together, it will trigger the submission logic only after all fields passed the validation schema. 

Basically, we need to call the `execute` method, update the recaptcha value and submit the form with the same trigger event. My problem is exactly that: I'm having to use two events (one for the `execute` method and update the recaptcha + one to submit the form).

Check this sandbox: https://codesandbox.io/s/keen-mahavira-ont8z?file=/src/App.js

As you can see, the form is submitted only with the second click in the submit button.

**Comment by OdifYltsaeb** [comment]
I think you got your answer here: https://stackoverflow.com/questions/65901771/how-to-make-invisible-react-google-recaptcha-formik-and-yup-work-together or did it not work?

**Comment by MuriloScarpaSitonio** [comment]
@OdifYltsaeb Yeah, it works!
